components/src/core/score-card/scorecard.component.scss

```diff
@@ -42,6 +42,12 @@
     .pxb-action-items-wrapper {
         text-overflow: unset;
         overflow: unset;
+       ::ng-deep > * {
+            cursor: pointer;
+            height: 32px;
+            width: 32px;
+            text-align: center;
+        }
```

Comment:
It would appear we didn't actually make this click-target update in react (only react native). The spacing is off a little still. The final positioning is supposed to be:
- Right edge of the rightmost icon (24px) is 16px from the right edge of the card
- 12px between icons (from the right edge of the left icon viewbox to the left edge of the right icon viewbox)

In RN, we achieved the larger touch targets using negative margins:
- icon wrapper height/width 40px with 8px padding and -8px margin
- icons size 24px

I don't know if this is crucial for web components. If we want to match react, we can just do a 12px margin between the icons.

Reply:
Ok I've updated the icons to be 24x24 and fixed the right padding on the 'action-items' container.  Now the card padding should be fine.

Reply:
did you push those changes?

Reply:
Yep, just pushed them out.  With more changes, updating PR desc now.